TW 715 Update Postman CLI governance options (#4320)
postmanlabs/postman-docs: src/pages/docs/postman-cli/postman-cli-options.md

postman api lint my-definition-file.json
postman api lint 8854915-bb7236b2-536e-4bdc-bfa2-fbe2fe1941eb
```

#### Options

Option | Details
--- | ---

> `--fail-severity [severity]`, `-F` | Specifies the severity level at which to trigger a failure exit code. The options, in order of lowest to highest severity, are `HINT`, `INFO`, `WARN`, and `ERROR` (default).

Reviewer: level at which (or above)

Reviewer: we hace decided to use '-f' instead of '-F'

> `--fail-severity [severity]`, `-F` | Specifies the severity level at which to trigger a failure exit code. The options, in order of lowest to highest severity, are `HINT`, `INFO`, `WARN`, and `ERROR` (default).

Author: ```suggestion
`--fail-severity [severity]`, `-F` | Triggers an exit failure code for rule violations at or above the specified severity level. The options, in order of lowest to highest severity, are `HINT`, `INFO`, `WARN`, and `ERROR` (default).
```

> `--fail-severity [severity]`, `-F` | Triggers an exit failure code for rule violations at or above the specified severity level. The options, in order of lowest to highest severity, are `HINT`, `INFO`, `WARN`, and `ERROR` (default).

Author: @chinmayjoshi Does this rewording work?

Reviewer: lgtm

Reviewer 2: @chinmayjoshi I believe we changed this to have smallcase -f. Let's update this @petewitcher

> `--fail-severity [severity]`, `-F` | Triggers an exit failure code for rule violations at or above the specified severity level. The options, in order of lowest to highest severity, are `HINT`, `INFO`, `WARN`, and `ERROR` (default).

Author: ```suggestion
`--fail-severity [severity]`, `-f` | Triggers an exit failure code for rule violations at or above the specified severity level. The options, in order of lowest to highest severity, are `HINT`, `INFO`, `WARN`, and `ERROR` (default).
```
